Root/JetSelector.cxx

```diff
@@ -155,182 +46,164 @@ JetSelector& JetSelector::setSystematic(const NtSys::SusyNtSys &s)
 //----------------------------------------------------------
 bool JetSelector::isBaselineJet(const Jet* jet)
 {
-    if(m_2lep || m_2lepWH || m_3lep) {
-        return (jet->Pt() > JET_MIN_PT_BASELINE);
-    } else {
-        bool passEta = (fabs(jet->Eta()) < JET_MAX_ETA);
-        bool passPt  = (jet->Pt() > JET_MIN_PT_BASELINE);
-        return (passEta && passPt);
-    }
+    return (jet->Pt() > 20.0);
 }
 //----------------------------------------------------------
 bool JetSelector::isSignalJet(const Jet* jet)
 {
-    JetSelector::check();
-
-    bool pass = false;
-    if(jet) {
-        if(m_2lep || m_2lepWH) {
-            pass = (isCentralLightJet(jet) || isCentralBJet(jet) || isForwardJet(jet));
-        }
-        else if(m_3lep || m_SS3L) {
-            pass = ((jet->Pt() > JET_MIN_PT_SIGNAL)
-                     && (fabs(jet->Eta()) < JET_MAX_ETA)
-                     && jetPassesJvtRequirement(jet));
-        }
-    } // if(jet)
-
-    return pass;
+    return (isCentralLightJet(jet) ||
+            isCentralBJet(jet) ||
+            isForwardJet(jet));
 }
 //----------------------------------------------------------
 bool JetSelector::isBJet(const Jet* jet)
 {
-    ///////////////////////////////
-    // pT requirement
-    ///////////////////////////////
-    if(jet->Pt() < 20.0)    return false;
-
-    ///////////////////////////////
-    // eta requirement
-    ///////////////////////////////
-    if(fabs(jet->Eta()) > 2.5) return false;
-
-    ///////////////////////////////
-    // jvt requirement
-    ///////////////////////////////
-    if(!jetPassesJvtRequirement(jet)) return false;
-
-    ///////////////////////////////
-    // mv2c20 cut
-    ///////////////////////////////
-    if(jet->mv2c20 < JET_MV2C20_77) return false;
-
-    return true;
+    return ((jet->Pt()        > 20.0   ) &&
+            (fabs(jet->Eta()) <  2.5   ) &&
+            (jet->mv2c20      > -0.4434) && // 77% eff
+            (passJvt(jet)));
 }
 //----------------------------------------------------------
 bool JetSelector::isCentralLightJet(const Jet* jet)
 {
-    JetSelector::check();
-
-    //////////////////////////////
-    // pt requirement
-    //////////////////////////////
-    if(jet->Pt() < JET_MIN_PT_L20)       return false;
-
-    //////////////////////////////
-    // detector eta requirement
-    //////////////////////////////
-    if(fabs(jet->detEta) > JET_MAX_ETA)  return false;
-
-    //////////////////////////////
-    // JVT requirement
-    //////////////////////////////
-    if(!jetPassesJvtRequirement(jet))    return false;
-
-    //////////////////////////////
-    // MV2c20 requirement
-    //   (a la SUSYTools)
-    //////////////////////////////
-    if(jet->bjet)                        return false;
```

Comment:
@dantrim was this `jet->bjet` intentional? in the new implementation I used `isBjet(jet)`
instead, so that it follows the b-tag operating point of the analysis.